rust/private/rustc.bzl

```
@@ -53,6 +53,11 @@ ErrorFormatInfo = provider(
     fields = {"error_format": "(string) [" + ", ".join(_error_format_values) + "]"},
 )
 
+OutputDiagnosticsInfo = provider(
+    doc = "Save json diagnostics form rustc",
```

```suggestion
    doc = "Save json diagnostics from rustc",
```
Can you also be more descriptive here on what "json diagnostics" is? Or reference docs that describes it more?